superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -328,16 +328,20 @@ class SliceHeaderControls extends React.PureComponent<
           {t('Download as image')}
         </Menu.Item>
 
-        {this.props.supersetCanCSV && (
-          <Menu.Item key={MENU_KEYS.EXPORT_CSV}>{t('Export CSV')}</Menu.Item>
-        )}
-        {isFeatureEnabled(FeatureFlag.ALLOW_FULL_CSV_EXPORT) &&
+        {this.props.slice.viz_type !== 'filter_box' &&
```

Comment:
Future improvement idea: maybe we should have a property `enableExportCSV` on `ChartMetadata` which defaults to `true`